frontend_v2/src/app/components/challenge/challenge.component.ts

```diff
@@ -51,6 +51,16 @@ export class ChallengeComponent implements OnInit {
    */
   isParticipated = false;
 
+  /**
+   * Is Forum enabled in Challenge
+   */
+  isForumEnabled;
```

Comment:
Either initialize them or at-least give them a type (same for the variable below).